site_scons/components/__init__.py

```diff
@@ -302,16 +302,14 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['cd dpdk; '                            \
-                          'git fetch; '                          \
-                          'git checkout origin/spdk-19.11.6',    \
-                          './configure --prefix="$SPDK_PREFIX"'                \
-                          ' --disable-tests --without-vhost --without-crypto'  \
-                          ' --without-pmdk --without-vpp --without-rbd'        \
-                          ' --with-rdma --with-shared'                         \
-                          ' --without-iscsi-initiator --without-isal'          \
-                          ' --without-vtune', 'make $JOBS_OPT', 'make install',
-                          'cp dpdk/build/lib/* "$SPDK_PREFIX/lib"',
+                commands=['./configure --prefix="$SPDK_PREFIX"'                \
+                          ' --disable-tests --disable-unit-tests '             \
+                          ' --disable-examples --disable-apps --without-vhost '\
+                          ' --without-crypto --without-pmdk --without-rbd '    \
+                          ' --with-rdma --without-iscsi-initiator '            \
+                          ' --without-isal --without-vtune --with-shared',
```

Comment:
I think we should move in the direction of unifying the build by having a separate dpdk component here so we aren't following a completely different process for RPM builds

Reply:
recorded as a follow-up task in https://jira.hpdd.intel.com/browse/DAOS-7960